pom.xml

```diff
@@ -106,6 +107,7 @@ Wisconsin-Madison.</license.copyrightOwners>
 		<dependency>
 			<groupId>org.scijava</groupId>
 			<artifactId>scijava-common</artifactId>
+			<version>2.64.1-SNAPSHOT</version>
```

Comment:
Also, it is better to use `<scijava-common.version>2.64.1-SNAPSHOT</scijava-common.version>` in the `<properties>` instead of hardcoding the `<version>` in the `<dependencies>` section. The former allows overriding the version in various circumstances such as the SciJava melting pot script, whereas the latter bakes in the version into the dependency hierarchy in an immutable way.